app/views/vacancies/_vacancy.html.erb

```diff
@@ -1,4 +1,5 @@
 <section class="vacancy-item ">
+  <%= link_to  vacancy_path(vacancy) do %>
   <div class="row">
```

Comment:
also, it is missing the indentation here.
ident one level (2 spaces) all things inside `link_to`.

Reply:
and there is 2 spaces after `link_to`. Make it with only one space plz

`link_to vacancy_path`...